### PR TITLE
Feature --ignore--inode

### PR DESCRIPTION
**Pull request by enkore**
Solves the issue with pull backups taking ages, because sshfs does not forward inode numbers from the server[1].

[1] I have not researched why, I guess it is a fundamental problem of sorts - if it were a simple "fix" they would just do it, I suppose.

### CODE REVIEW

File: borg/archiver.py [hunk 1]
```diff
@@ -1018,6 +1025,9 @@ def build_parser(self, args=None, prog=None):
                                type=ChunkerParams, default=CHUNKER_PARAMS,
                                metavar='CHUNK_MIN_EXP,CHUNK_MAX_EXP,HASH_MASK_BITS,HASH_WINDOW_SIZE',
                                help='specify the chunker parameters. default: %d,%d,%d,%d' % CHUNKER_PARAMS)
+        subparser.add_argument('--ignore-inode', dest='ignore_inode',
+                               action='store_true', default=False,
```
Comment (enkore) [inline borg/archiver.py]: I think `action='store_true', default=False,` is redundant, but it's always used here. Maybe something for a little cleanup session.

**Comment by enkore** [comment]
stapling #628

**Comment by piotrjurkiewicz** [comment]
This feature is useful also on Virtualbox shared folders (vboxsf filesystem). Inodes numbers are not constant between mounts. It would be good to mention this in help as well.

**Comment by ThomasWaldmann** [comment]
There are likely quite some filesystem that do not have stable inode numbers, so I guess we rather just document our option and what it does than trying to enumerate them all (documenting that a fs does not have stable inodes is in scope of the fs documentation).

**Comment by enkore** [comment]
Q: Also push this into 1.0.1, to "fix" the performance issues with sshfs and others for 1.0.x users?

**Comment by ThomasWaldmann** [comment]
@enkore yes, makes sense.

**Pull request merged**